tsconfig.json

```diff
@@ -1,12 +1,21 @@
 {
-	"extends": "@sindresorhus/tsconfig/tsconfig.json",
-	"include": [
-		"examples",
-		"source",
-		"test"
-	],
-	"compilerOptions": {
-		"module": "CommonJS",
-		"outDir": "dist"
-	}
+  "compilerOptions": {
+    "charset": "utf8",
+    "forceConsistentCasingInFileNames": true,
+    "newLine": "lf",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "noUnusedParameters": true,
+    "rootDir": "./src/",
+    "strict": true,
+    "declaration": true,
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "outDir": "./out/",
+    "skipLibCheck": true,
+    "target": "es2019"
+  },
+  "include": [
+    "src/"
+  ]
```

Comment:
I think only `deno2node` requires this file, correct? So since we moved to `dnt` now, it should be deleted.

Reply:
Ah, yes.